lesson4/backend28.js

```diff
@@ -5,14 +5,20 @@ app.listen(3618);
 app.use(express.static('public'));
 const bodyParser = require('body-parser');
 app.use(bodyParser.json({limit: '500mb'}));
+const gm = require('gm');
 
 app.post('/send',(req,res)=>{
   const firstData = req.body.name;
   const secondData = req.body.comment;
   const pic = req.body.newPicData;
-  fs.appendFile('/home/kinue/garageScript/lesson4/public/storing.txt',`${firstData} : ${secondData}\n`)
+  const storingTxt = '/home/kinue/garageScript/lesson4/public/storing.txt';
+  const memePath = '/home/kinue/garageScript/lesson4/public/MemePhotos/' + firstData + '.png';
+  
+  fs.appendFile(storingTxt,`${firstData} : ${secondData}\n`)
 
-  fs.writeFile('/home/kinue/garageScript/lesson4/public/selfies.png',`${pic}`, 'base64') 
-  res.send('Pic taken');
+  fs.writeFile(memePath, `${pic}`, 'base64', (err) =>{
```

Comment:
what do you mean by `${pic}`? My guess is you don't understand how expression fully works. Talk to me if you are confused about this

Reply:
ok, I'll explain this one as well once you get here.